### DISCUSSION TITLE
Start window location

### DISCUSSION THREAD

**Discussion by kleyer** [elfmz/far2l#1105] [Q&A]
Hello 

Is it possible to restore window initial location on far2l app start?
it stores window size but not the location for some reason (at least on MacOS).

**Comment by DEF29** [comment]
Hello

I have same problem and I temporarily use Moom program by hotkey to center FAR2L window after he start.

See my Discussion post:
https://github.com/elfmz/far2l/discussions/1364

I hope window position problem will be fixed at future...
